packages/content/test/attributes/Website.test.ts

```diff
@@ -0,0 +1,37 @@
+import { ParsingError } from "@js-soft/ts-serval";
+import { Website } from "../../src";
+
+describe("Test valid URLs", () => {
+    const validUrls = [
+        "www.google.com",
+        "https://inwänd.it",
+        "http://inwind.it",
+        "https://enmeshed.de/blog/meilenstein-enmeshed-als-komponente-ablage-in-mein-bildungsraum-geht-in-die-testphase-der-beta-version/",
+        "www.foo.www.www.enmeshed.eu",
+        "https://example.org:8080/mein/ordner/bericht"
+    ];
+
+    test.each(validUrls)("URL %s is recognized as valid", (url) => {
+        const validWebsite = Website.from({ value: url });
+        expect(validWebsite.value.toString()).toBe(url);
+    });
+});
+
+describe("Test invalid URLs", () => {
+    const invalidUrls = ["google-.de", "www.google", "www.-google", "https://inwind.test it"];
+
```

Comment:
Add a test for an empty URL.

Reply:
Should we also accept an empty emailaddress?

Reply:
I meant to _reject_ empty strings :-) Same for email addresses, too.